pymemcache/client/base.py

```diff
@@ -72,7 +72,7 @@ def _check_key(key, key_prefix=b''):
         except UnicodeEncodeError:
             raise MemcacheIllegalInputError("No ascii key: %r" % (key,))
     key = key_prefix + key
-    if b' ' in key:
+    if b' ' in key or b'\r\n' in key:
```

Comment:
Are we only worried about `\r\n` (and `\r` and `\n` are individually acceptable)?